core/src/torii/mod.rs

```diff
@@ -219,6 +228,12 @@ impl<W: WorldTrait> Torii<W> {
             .with(warp::trace::request())
             .recover(Torii::<W>::recover_arg_parse);
 
+        tokio::spawn(async move {
+            start_status(Arc::clone(&state))
+                .await
+                .wrap_err("Failed to start status service")
+        });
```

Comment:
There should be something done with an error. Smth like this can be done:
```suggestion
        tokio::spawn(async move {
            let res = start_status(Arc::clone(&state))
                .await
                .wrap_err("Failed to start status service");
            if let Err(err) = res {
                iroha_logger::error!(%error);
            }
        });
```